src/common/components/cards/urls-card-row.tsx

```diff
@@ -1,26 +1,36 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-import { NewTabLink } from 'common/components/new-tab-link';
 import * as React from 'react';
 
-import { CardRowProps } from 'common/configs/unified-result-property-configurations';
+import { CardRowDeps, CardRowProps } from 'common/configs/unified-result-property-configurations';
 import { NamedFC } from 'common/react/named-fc';
+import { LinkComponentType } from 'common/types/link-component-type';
 import { SimpleCardRow } from './simple-card-row';
+import * as styles from './urls-card-row.scss';
+
+export type UrlsCardRowDeps = CardRowDeps & {
+    LinkComponent: LinkComponentType;
```

Comment:
I think this is actually going to need to go into `CardRowDeps`, since that's what the parent container components are going to be composing with; the individualized rows contain their own type identifier and propertyData, so those can be coupled here, but the deps come from the parent container independent of knowing the row types